reversestring-MVV/prompts.md

```
@@ -0,0 +1,56 @@
+    - Validar que no se pueda hacer una cadena de texto vacía.
+    - Validar que no se pueda hacer una cadena de texto con menos de 3 caracteres.
+    - El output de la cadena invertida debe ser mostrado en tiempo real, con cada modificacion que se haga en el input.
+    - Usa programación funcional para invertir la cadena de texto.
+
+#salida
+    - El proyecto debe ser un archivo html con nombre index.html que se pueda ejecutar en el navegador.
+    - El proyecto debe ser un archivo js con nombre script.js que se pueda ejecutar en el navegador.
+    - El proyecto debe ser un archivo css con nombre style.css que se pueda ejecutar en el navegador.
+#restriciones
+    - No debes usar estilos inline
+#ejemplos de salida
```

markdownlint-cli2 (0.18.1)

No space after hash on atx style heading

(MD018, no-missing-space-atx)